codegen/smithy-aws-swift-codegen/src/main/kotlin/software/amazon/smithy/aws/swift/codegen/AWSHttpProtocolServiceClient.kt

```diff
@@ -45,7 +47,15 @@ class AWSHttpProtocolServiceClient(
                     ConfigProperty("authSchemeResolver", SmithyHTTPAuthAPITypes.AuthSchemeResolver, authSchemeResolverDefaultProvider)
                 }
                 "authSchemes" -> {
-                    ConfigProperty("authSchemes", SmithyHTTPAuthAPITypes.AuthSchemes.toOptional(), authSchemesDefaultProvider)
+                    ConfigProperty("authSchemes", SmithyHTTPAuthAPITypes.AuthSchemes.toOptional(), AWSAuthUtils(ctx).authSchemesDefaultProvider)
+                }
+                "bearerTokenIdentityResolver" -> {
+                    ConfigProperty(
+                        "bearerTokenIdentityResolver",
+                        SmithyIdentityTypes.BearerTokenIdentityResolver.toGeneric(),
+                        { it.format("\$N()", AWSSDKIdentityTypes.DefaultBearerTokenIdentityResolverChain) },
+                        true
+                    )
```

Comment:
The default value for `bearerTokenIdentityResolver` field of the client config was `StaticBearerTokenIdentityResolver` in `smithy-swift`, but over here in `aws-sdk-swift`, it has to be the default chain instead. Hence this override.